packages/overlay/src/HoverController.ts

```diff
@@ -184,9 +229,9 @@ export class HoverController extends InteractionController {
     }
 
     override initOverlay(): void {
-        if (!this.abortController) {
-            return;
-        }
+        // Clean up listeners if they've already been bound
+        this.abortController?.abort();
+        this.abortController = new AbortController();
         const { signal } = this.abortController;
```

Comment:
I think we should not be aborting here, all the target element's event listeners are removed as soon as the overlay is initialized.